publication.go

```diff
@@ -171,7 +171,7 @@ func (p *Publication) Reply(response *Publication) error {
 
 	switch {
 	case p.replyCh == nil:
-		return errors.New("no response required for publication")
+		return errors.New("no response required for publication or took too long to call Reply")
```

Comment:
I could return a specific message for this by adding a new un-exported boolean attribute to `Publication` - `timedOut`

Reply:
Donezo ✅